tests/ComposerImporterTest.php

```diff
@@ -160,6 +160,46 @@ public function testPSR4ImportNoTrailingSlash(Parser $parser): void {
     );
   }
 
+  /**
+   * @dataProvider getParsers
+   */
+  public function testPSR4ImportWithoutPrefix(Parser $parser): void {
+    $root = \realpath(__DIR__.'/fixtures/psr-4');
+    $composer = $root.'/composer.json';
+    $this->assertTrue(\file_exists($composer));
```

Comment:
please use `Facebook\FBExpect\expect()` for new tests; a migration will be needed soon.

In this case, you need `use function` at the top, then `expect(\file_exists($composer))->toBeTrue()`